Confirmed, and thank you for chasing it down to the actual cause rather than just re-running — I pulled the job logs to check before agreeing, because the mixed pass/fail pattern within a single run initially read to me like a real failure. It is not. `Integration tests` (job 95415250013) dies at `Set up job`:

```
Download action repository 'step-security/harden-runner@b09bb98e...'
Failed to download action ... 429 (Too Many Requests)
Failed to download action ... 429 (Too Many Requests)
##[error]Failed to download archive ... after 3 attempts.
```

The reason it hits some jobs and not others is that codeload was rate-limiting rather than fully down, so each job rolled its own dice on the three retries. Your read was right and mine was wrong on first look.

It appears to have cleared: jobs starting from about 16:20 are pulling the action fine (the install smokes on #4099 pass at 16:20). So a re-run should be clean.

One caveat before you treat this PR as green-modulo-infra: `Diff coverage report` also failed here, in the same window, and I could not tell from the timing alone whether that was the same setup abort or a genuine coverage shortfall on the new lines. Worth a look at that specific job after the next clean run — if it is real, it is a small fix and I would rather you catch it now than after a re-run makes everything else green.

I tried to kick the re-run for you and the API refused it for my token (`403 Resource not accessible by integration`), so it needs to come from you or from a push. Sorry — that is the one part of this I could not just clear.

---
_Generated by [Claude Code](https://claude.ai/code)_

--- a/.github/workflows/adapter-conformance-canary.yml
+++ b/.github/workflows/adapter-conformance-canary.yml
@@ -161,17 +161,6 @@ jobs:
           fi
           echo "regressions=$rc" >> "$GITHUB_OUTPUT"
 
-      - name: Verify last-green projects the receipts
-        # The run writes last_green.json and the docs table from one pass, so
-        # checking those two against each other proves only that they share a
-        # generator. This re-reads the receipts off disk in a fresh process and
-        # re-derives the projection, which is the step that was never checked
-        # (#3940). Runs before the upload so a bad projection is caught while
-        # the receipts are still local.
-        run: |
-          set -euo pipefail
-          uv run python scripts/adapter_canary.py --verify-projection             --out-dir .sdd/runtime/adapter-canary
-
       - name: Save canary state
         if: always()
         uses: actions/cache/save@55cc8345863c7cc4c66a329aec7e433d2d1c52a9 # v6.1.0
@@ -218,6 +207,17 @@ jobs:
 
           Workflow run: ${{ github.server_url }}/${{ github.repository }}/actions/runs/${{ github.run_id }}"
           done
+
+      - name: Verify last-green projects the receipts
+        # Must stay after `Run canary matrix` (and `--update-docs`) because it
+        # re-reads the receipts off disk in a fresh process and re-derives the
+        # projection against the freshly updated working tree (#3940).
+        # Placed after regression issue opening so a projection mismatch does
+        # not suppress threshold-crossing regression issue creation.
+        run: |
+          set -euo pipefail
+          uv run python scripts/adapter_canary.py --verify-projection \
+            --out-dir .sdd/runtime/adapter-canary
 
       - name: Propose last-green regeneration PR
         env:

--- a/docs/adapters/conformance-canary.md
+++ b/docs/adapters/conformance-canary.md
@@ -56,8 +56,9 @@ passes.
 `verify_last_green_projection` closes that gap by re-reading the receipts
 and asking whether they produce the committed rows. The nightly workflow
 runs it as its own step (`--verify-projection`), in a fresh process,
-after the matrix and **before** the receipts are uploaded, so a bad
-projection is caught while the receipts are still on local disk.
+after `Open threshold-crossing regression issues` and before PR proposal,
+so a projection mismatch is caught while receipts are still on local disk
+without suppressing threshold-crossing regression issue creation.
 
 It checks both directions for the adapters actually in play: every
 passing receipt must have a row carrying that receipt's digest and this

